src/app/components/home-page/home-page.component.ts

```diff
@@ -17,9 +17,14 @@ export class HomePageComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.userId = this.route.snapshot.paramMap.get("id");
+    this.userId = this.route.snapshot.paramMap.get('id');
     this.userService.getUser(this.userId).subscribe(data => {
       this.user = data;
+      sessionStorage.setItem('id', this.user.id);
```

Comment:
nazwa powinna byc unikalna bo itemu bo inna stronka będzie mogła Ci napisać. poza tym wedle naszych wymagań lepiej gdy to bedzie windowStorage.

Reply:
Tak pamietam gdy mówiles o tym windowStorage i nic takiego nie moge znalzc jedynie widze localStorage i ten session

Reply:
Ew z tego co czytam moglbym localStorage  i w onDestory czyscic ale to raczej bezcelowe by bylo

Reply:
pomyliło mi się, sessionStorage to jest to co potrzebujemy